docusaurus/docs/metadata.md

```diff
@@ -207,6 +209,30 @@ For data source plugins. There is a query options section in the plugin's query
 | `maxDataPoints` | boolean | No       | For data source plugins. If the `max data points` option should be shown in the query options section in the query editor. |
 | `minInterval`   | boolean | No       | For data source plugins. If the `min interval` option should be shown in the query options section in the query editor.    |
 
+## roles
+
+List of RBAC roles defined by the plugin and their default assignments to basic roles (`Viewer`, `Editor`, `Admin`, `Grafana Admin`).
+
+Requires Grafana version 9.4.0 or later. Currently, this is behind the `accessControlOnCall` feature toggle.
+
+### Properties
+
+| Property | Type            | Required | Description                                                                                           |
+| -------- | --------------- | -------- | ----------------------------------------------------------------------------------------------------- |
+| `role`   | [object](#role) | **Yes**  | RBAC role definition to group related RBAC permissions on the plugin.                                 |
+| `grant`  | string[]        | No       | Default assignments of the role to Grafana basic roles (`Viewer`, `Editor`, `Admin`, `Grafana Admin`) |
+
+### role
+
+A role groups your plugin's related RBAC permissions (ex: `Projects Admin` would group permissions to create, read, write and delete projects).
+The RBAC actions defined in your role must start with your plugin `id` (ex: `grafana-test-app.projects:read`).
+
+| Property      | Type                     | Required | Description                    |
+| ------------- | ------------------------ | -------- | ------------------------------ |
+| `name`        | string                   | **Yes**  | Name of the role.              |
+| `description` | string                   | No       | Describes the aim of the role. |
+| `permissions` | [object](#permissions)[] | No       | RBAC permission on the plugin. |
+
```

Comment:
**Nit:** maybe it would make sense to include a small example below for a hypothetical use-case?

Reply:
I think it makes sense.
I'm trying to write an example plugin: https://github.com/grafana/grafana-plugin-examples/pull/242
I'll add it to the roles section.
```md
To try this feature out, follow this [example](https://github.com/grafana/grafana-plugin-examples/blob/main/examples/app-with-rbac/README.md)
```
Wdyt, should I put this PR on hold and wait for the example plugin to be merged?
Or should I merge this one, we track that comment in an issue, then add the link?